lib/matplotlib/axes/_base.py

```diff
@@ -2075,12 +2235,13 @@ def _update_patch_limits(self, patch):
 
     def add_table(self, tab):
         """
-        Add a `~.Table` to the axes' tables; return the table.
+        Add a `~.Table` to the Axes; return the table.
         """
+        self._deprecate_noninstance('add_table', mtable.Table, tab=tab)
         self._set_artist_props(tab)
-        self.tables.append(tab)
+        self._children.append(tab)
         tab.set_clip_path(self.patch)
-        tab._remove_method = self.tables.remove
+        tab._remove_method = self._children.remove
```

Comment:
I thought this `_remove_method` stuff was to allow automatically removing from the correct list. Is there no way we could eliminate this stuff too?

Reply:
We could maybe add a default? But `Artist.remove` calls this, and it also is used for, e.g., removing Artists added directly to Figures, or the Legend remove, which are not linked to these lists.

Reply:
Ugh, I forget about artists added elsewhere. Ok, probably can't do away with `_remove_method`.